nilearn/glm/thresholding.py

```diff
@@ -217,6 +219,7 @@ def threshold_stats_img(
     height_control : :obj:`str`, or None, default='fpr'
         False positive control meaning of cluster forming
         threshold: None|'fpr'|'fdr'|'bonferroni'
+        Only `None` and `'bonferroni'` supported for SurfaceImages.
```

Comment:
There is no technical reason not to support the others.

Reply:
indeed: i thought fdr and fpr had some volume specific things to them